docs/source/conf.py

```diff
@@ -29,13 +29,13 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    "sphinx.ext.autodoc",
     "sphinx.ext.coverage",
     "sphinx.ext.napoleon",
     "sphinx.ext.mathjax",
     "sphinx.ext.viewcode",
     "sphinx_autodoc_typehints",
     "sphinx_rtd_theme",
+    "sphinx.ext.autosummary",
```

Comment:
This package enables automatically building generated source files remotely on readthedocs.io upon a push to the master branch.